contract/src/flows/test/basic-flows.contract.test.js

```diff
@@ -0,0 +1,88 @@
+import { test as anyTest } from '@agoric/zoe/tools/prepare-test-env-ava.js';
```

Comment:
Let's not put `test/` and `tools/` code under `src/`, please.

Reply:
Done!